src/chi-vue/src/components/column-customization/ColumnCustomization.tsx

```diff
@@ -204,11 +204,6 @@ export default class ColumnCustomization extends Vue {
       });
     }
   }
-
-  beforeMount() {
-    this._chiMajorVersion = detectMajorChiVersion();
```

Comment:
This removes the lifecycle hook but the declaration of `_chiMajorVersion` is still there. Please check line 34.